fenix/app/build.gradle

```diff
@@ -606,6 +606,7 @@ dependencies {
     implementation FenixDependencies.androidx_lifecycle_viewmodel
     implementation FenixDependencies.androidx_core
     implementation FenixDependencies.androidx_core_ktx
+    implementation FenixDependencies.androidx_tracing
```

Comment:
Why `implementation`? This would go into our release APK. Shouldn't this be in maybe `testImplementation`? I think that would make sense since the AndroidJUnitTestRunner is used there. Unclear to me though why that is needed in the UI tests though.